__tests__/auth.test.ts

```diff
@@ -1,40 +1,42 @@
-import createJWKSMock from 'mock-jwks';
-import Auth from '../auth';
+import createJWKSMock from 'mock-jwks'
+import { createCheckScopesAndResolve } from '../auth'
 
-import { APIGatewayProxyEvent } from 'aws-lambda';
+import { APIGatewayProxyEvent } from 'aws-lambda'
+
+const TOKEN_ISSUER = 'https://test-app.com/'
 
 describe('Given an event with an [INCORRECT] scope', () => {
   it('Returned - You are not authorized!', async () => {
-    // const jwksMock = await createJwksContext();
-    process.env.JWKS_URI = 'https://test-app.com/.well-known/jwks.json';
-    process.env.TOKEN_ISSUER = 'https://test-app.com/';
-    process.env.AUDIENCE = 'https://test-app.com/test/';
-    const jwksMock = createJWKSMock('https://test-app.com/');
-    await jwksMock.start();
+    const jwksMock = createJWKSMock(TOKEN_ISSUER)
+    await jwksMock.start()
+
+    const checkScopesAndResolve = createCheckScopesAndResolve({
+      jwksUri: 'https://test-app.com/.well-known/jwks.json',
+      issuer: TOKEN_ISSUER,
+      audience: 'https://test-app.com/test/',
+    })
```

Comment:
This is how you can configure the context for your function. Later you can get these values from the environment but you should not set environment variables in test. This might have all kinds of undesired side effects.